22:46:11,176 WARN  eu.stratosphere.nephele.configuration.GlobalConfiguration     - Cannot load configuration: unknown element plugins
22:46:11,212 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Reading location of job manager from configuration
22:46:11,222 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Determined address of job manager to be cloud-11.dima.tu-berlin.de/130.149.21.15:6002
22:46:11,231 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Announcing connection information cloud-26 to job manager
22:46:11,297 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server Responder: starting
22:46:11,297 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server listener on 6122: starting
22:46:11,298 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server handler 0 on 6122: starting
22:46:12,419 INFO  eu.stratosphere.nephele.taskmanager.bufferprovider.GlobalBufferPool  - Initialized global buffer pool with 32768 buffers with a size 32768 bytes each
22:46:12,438 INFO  eu.stratosphere.nephele.taskmanager.bytebuffered.ByteBufferedChannelManager  - Initialized byte buffered channel manager with sender-side spilling disabled
22:46:12,448 INFO  eu.stratosphere.nephele.instance.HardwareDescriptionFactory   - Found Tenured Gen pool (max: 24051843072, used: 0)
22:46:12,448 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Initializing memory manager with 22528 megabytes of memory
22:46:46,189 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Build Triads (10/25)
22:46:46,193 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Build Triads (10/25)
22:46:47,858 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (75/200)
22:46:47,858 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (75/200)
22:46:47,860 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (75/200)
22:46:47,861 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (75/200)
22:46:48,464 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (73/200)
22:46:48,465 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (73/200)
22:46:48,465 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (73/200)
22:46:48,466 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (73/200)
22:46:48,497 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (80/200)
22:46:48,497 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (80/200)
22:46:48,499 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (80/200)
22:46:48,500 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (80/200)
22:46:48,521 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (77/200)
22:46:48,521 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (77/200)
22:46:48,523 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (77/200)
22:46:48,524 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (77/200)
22:46:48,575 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (79/200)
22:46:48,575 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (79/200)
22:46:48,818 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (79/200)
22:46:48,819 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (79/200)
22:46:48,865 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (76/200)
22:46:48,865 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (76/200)
22:46:48,867 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (76/200)
22:46:48,869 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (76/200)
22:46:52,583 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (78/200)
22:46:52,583 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (78/200)
22:46:52,585 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (78/200)
22:46:52,585 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (78/200)
22:46:53,313 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (74/200)
22:46:53,314 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (74/200)
22:46:53,315 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (74/200)
22:46:53,317 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (74/200)
22:47:24,134 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Finished PACT code.: Build Triads (10/25)
22:47:24,134 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from RUNNING to FINISHING for task Build Triads (10/25)
22:49:25,400 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-3655204604942431391_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:39186 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:25,401 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-3655204604942431391_1722 bad datanode[0] 130.149.21.30:45020
22:49:25,402 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-3655204604942431391_1722 in pipeline 130.149.21.30:45020, 130.149.21.34:45020, 130.149.21.60:45020: bad datanode 130.149.21.30:45020
22:49:26,195 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-5637283328712417129_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:39195 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:26,196 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-5637283328712417129_1722 bad datanode[0] 130.149.21.30:45020
22:49:26,196 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-5637283328712417129_1722 in pipeline 130.149.21.30:45020, 130.149.21.26:45020, 130.149.21.21:45020: bad datanode 130.149.21.30:45020
22:49:29,153 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1764347952016783573_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:39198 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:29,153 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1764347952016783573_1722 bad datanode[0] 130.149.21.30:45020
22:49:29,154 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1764347952016783573_1722 in pipeline 130.149.21.30:45020, 130.149.21.22:45020, 130.149.21.64:45020: bad datanode 130.149.21.30:45020
22:49:29,642 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1421475303392509358_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:39184 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:29,643 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1421475303392509358_1722 bad datanode[0] 130.149.21.30:45020
22:49:29,644 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1421475303392509358_1722 in pipeline 130.149.21.30:45020, 130.149.21.25:45020, 130.149.21.19:45020: bad datanode 130.149.21.30:45020
22:49:40,500 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_7321962553348313838_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:39193 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:40,501 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_7321962553348313838_1722 bad datanode[0] 130.149.21.30:45020
22:49:40,502 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_7321962553348313838_1722 in pipeline 130.149.21.30:45020, 130.149.21.17:45020, 130.149.21.33:45020: bad datanode 130.149.21.30:45020
22:49:58,784 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_5033873691370285566_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:39202 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:58,785 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5033873691370285566_1722 bad datanode[0] 130.149.21.30:45020
22:49:58,786 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5033873691370285566_1722 in pipeline 130.149.21.30:45020, 130.149.21.3:45020, 130.149.21.22:45020: bad datanode 130.149.21.30:45020
22:50:04,279 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1994156433901416646_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:39200 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:04,280 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1994156433901416646_1722 bad datanode[0] 130.149.21.30:45020
22:50:04,281 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1994156433901416646_1722 in pipeline 130.149.21.30:45020, 130.149.21.64:45020, 130.149.21.21:45020: bad datanode 130.149.21.30:45020
22:50:05,217 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-9002472481688842064_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:39188 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:05,218 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-9002472481688842064_1722 bad datanode[0] 130.149.21.30:45020
22:50:05,219 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-9002472481688842064_1722 in pipeline 130.149.21.30:45020, 130.149.21.20:45020, 130.149.21.29:45020: bad datanode 130.149.21.30:45020
22:50:31,512 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-3655204604942431391_1741java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:55020 remote=/130.149.21.34:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:31,513 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-3655204604942431391_1741 bad datanode[0] 130.149.21.34:45020
22:50:31,514 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-3655204604942431391_1741 in pipeline 130.149.21.34:45020, 130.149.21.60:45020: bad datanode 130.149.21.34:45020
22:50:32,293 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-5637283328712417129_1745java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:34470 remote=/130.149.21.26:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:32,294 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-5637283328712417129_1745 bad datanode[0] 130.149.21.26:45020
22:50:32,294 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-5637283328712417129_1745 in pipeline 130.149.21.26:45020, 130.149.21.21:45020: bad datanode 130.149.21.26:45020
22:50:35,275 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1764347952016783573_1761java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:37776 remote=/130.149.21.22:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:35,277 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1764347952016783573_1761 bad datanode[0] 130.149.21.22:45020
22:50:35,277 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1764347952016783573_1761 in pipeline 130.149.21.22:45020, 130.149.21.64:45020: bad datanode 130.149.21.22:45020
22:50:35,744 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1421475303392509358_1767java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:55279 remote=/130.149.21.25:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:35,744 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1421475303392509358_1767 bad datanode[0] 130.149.21.25:45020
22:50:35,745 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1421475303392509358_1767 in pipeline 130.149.21.25:45020, 130.149.21.19:45020: bad datanode 130.149.21.25:45020
22:50:46,603 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_7321962553348313838_1787java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:44219 remote=/130.149.21.17:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:46,604 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_7321962553348313838_1787 bad datanode[0] 130.149.21.17:45020
22:50:46,605 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_7321962553348313838_1787 in pipeline 130.149.21.17:45020, 130.149.21.33:45020: bad datanode 130.149.21.17:45020
22:51:10,381 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1994156433901416646_1828java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:58668 remote=/130.149.21.64:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:10,383 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1994156433901416646_1828 bad datanode[0] 130.149.21.64:45020
22:51:10,383 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1994156433901416646_1828 in pipeline 130.149.21.64:45020, 130.149.21.21:45020: bad datanode 130.149.21.64:45020
22:51:11,313 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-9002472481688842064_1834java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:54016 remote=/130.149.21.20:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:11,314 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-9002472481688842064_1834 bad datanode[0] 130.149.21.20:45020
22:51:11,315 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-9002472481688842064_1834 in pipeline 130.149.21.20:45020, 130.149.21.29:45020: bad datanode 130.149.21.20:45020
22:51:27,945 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_5033873691370285566_1820java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.30:33691 remote=/130.149.21.3:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:27,947 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5033873691370285566_1820 bad datanode[0] 130.149.21.3:45020
22:51:27,947 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5033873691370285566_1820 in pipeline 130.149.21.3:45020, 130.149.21.22:45020: bad datanode 130.149.21.3:45020
